dmutils/timing.py

```diff
@@ -10,27 +11,6 @@
 SLOW_EXTERNAL_CALL_THRESHOLD = 0.25
 
 
-# General public conditions that may be useful
-def exceeds_slow_external_call_threshold(log_context):
-    """A public condition that will return True if the duration is above the threshold we have defined as acceptable for
-    calls to external services (e.g. Notify, Mailchimp, S3, etc)."""
-    return log_context['duration_real'] > SLOW_EXTERNAL_CALL_THRESHOLD
-
-
-def request_is_sampled(log_context):
-    """A public condition that returns True if the request has the X-B3-Sampled flag set in its headers. While this is
-    the default condition for logged_duration, exposing it publically allows it to be easily combined with other
-    conditions."""
-    return has_request_context() and getattr(request, "is_sampled", False)
-# End public conditions ----------------------
```

Comment:
I'm not clear why we lost these comments, but it seems to have made the diff much bigger than it needs to be (or something has).

Reply:
I moved this block down in the file. It does make for an ugly diff (sorry), but it feels better grouped at the bottom of the file.